osu.Game/OsuGame.cs

```diff
@@ -242,16 +242,19 @@ private bool globalHotkeyPressed(InputState state, KeyDownEventArgs args)
                     return true;
             }
 
-            if (state.Keyboard.ControlPressed)
+            if (currentScreen.UseSettingsOverlays)

```

Comment:
i don't think we want to do this. options are intended to be accessible at any point game-wide, including during gameplay if you wish.